src/Assertions/Assertions.jl

```diff
@@ -0,0 +1,28 @@
+module Assertions
+
+export activate_assertions,
+       deactivate_assertions,
+       are_assertions_enabled
+
+# source: https://discourse.julialang.org/t/defensive-programming-assert/8383/11
+
+# functions to (de)activate assertions
```

Comment:
Since these functions are exported and it is anyways good to have, we should add their docstrings with a short description about how to use them. We can add an Examples subsection with [the example in the comments](https://github.com/JuliaReach/LazySets.jl/pull/1732#issuecomment-535103718).

Reply:
we can add it later too, for me this PR is good to go.. (see https://github.com/JuliaReach/LazySets.jl/issues/1733)

Reply:
Agreeed, let us do it later.